packages/slice-machine/src/theme.ts

```diff
@@ -886,10 +886,6 @@ const AppTheme = (): Theme =>
         variant: "textStyles.heading",
         fontSize: 5,
       },
-      h3: {
```

Comment:
Removed that as it was causing a previous change I did bad. The blank slate title was too big because of that style being applied. As we don't want theme-ui styles + we don't have other h3 that we want to style with that, I removed it.